### ISSUE TITLE
Banner, failing contrast

### ISSUE THREAD

**Issue by WilliamKelley** [prenda-school/design-system#257]
Migrated from #149

### Design Feedback

cc: @ssbittle 

Three of the four variants are failing all WCAG 2.1 contrast ratio minimum for normal text. This is probably not an acceptable initial implementation.

![image](https://user-images.githubusercontent.com/25781782/130997588-b9275443-2408-4fee-953c-920eba5b1c59.png)
![image](https://user-images.githubusercontent.com/25781782/130997593-a3fd17bd-8f51-4c50-8d2c-b111894b8715.png)
![image](https://user-images.githubusercontent.com/25781782/130997597-5f9c591f-d763-4ac8-985a-14f55dac48ab.png)
![image](https://user-images.githubusercontent.com/25781782/130997599-8d070020-5fad-49c1-b24a-9836182f3cba.png)

**Comment by alecdanz** [comment]
Hey, @WilliamKelley thank you for bringing this up, we are in the process of revaluating our color usage and secondary brand colors (such as the ones used in these banners.) 

We should have foundational color updates figured out by next Wednesday and from there we can begin implementation of our new colors throughout design system components.

**Comment by alecdanz** [comment]
New [banners and colors improvements ](https://www.figma.com/file/y6Ca8BdJtQEd3Of1saRieF/components?node-id=665%3A2)have accounted for this.
